dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -340,6 +341,7 @@ export const Card = ({
 	trailTextSize,
 	trailTextColour,
 	podcastImage,
+	galleryCount,
```

Comment:
Rather than adding a `console.log` below to satisfy linting you could temporarily disable the check on this line:

```suggestion
	// eslint-disable-next-line @typescript-eslint/no-unused-vars -- Added in preparation for UI changes to display gallery count
	galleryCount,
```

Reply:
ah yes, i've applied this change